aaronsekisambu/my-personal-projects
# Background and gradients

## Description
- Styling the page using gradient and background images

## How to Manually Test
- Clone the repo and fetch all branches from the origin
- Check to `background-and-gradients`
- Run the index.html file
## Screenshots
